test/test_custom_repair_wheel.py

```diff
@@ -21,8 +21,7 @@
 name = f"spam-0.1.0-py2-none-{platform}.whl"
 dest = dest_dir / name
 dest_dir.mkdir(parents=True, exist_ok=True)
-if dest.exists():
-    dest.unlink()
+dest.unlink(missing_ok=True)
```

Comment:
This one won't work, because it was added in Python 3.8, and the repair script runs in the build, so runs on 3.6 and 3.7.

Reply:
Though it would be fine after #2282?

Reply:
I rebased on main and added this back in. Let's see if CI passes!